package.json

```diff
@@ -14,7 +14,7 @@
     "babel-minify-webpack-plugin": "0.3.1",
     "source-map-loader": "0.2.4",
     "ts-loader": "5.2.2",
-    "typescript": "3.1.3",
+    "typescript": "^3.5.1",
```

Comment:
What do you need this version of Typescript for?

Reply:
It was a dependency that got changed when I was trying to figure out why TSlint isn't working right now.

Reply:
Look into reverting this change. Upgrading the compiler is a major change.